libs/datamodel/parser-database/src/walkers/relation.rs

```diff
@@ -210,7 +204,13 @@ impl<'ast, 'db> InlineRelationWalker<'ast, 'db> {
             })
     }
 
-    pub(crate) fn back_relation_field(self) -> Option<RelationFieldWalker<'ast, 'db>> {
+    /// The contents of the `map: ...` argument of the `@relation` attribute.
```

Comment:
The comment makes it sound like this returns the FK name, but it is the fields dbname correct?

Reply:
Hmm I don't understand the difference (which is probably why my code is ambiguous in the first place :p)

Reply:
the field.explicit_mapped_name() in the code sounds like it could be the fields map attribute. Instead its the explicit foreign key name. Maybe just change the name of the method explicit_mapped_name -> explicit_fk_name ?

Reply:
Aah I understand better. My mental model was that foreign key was something we should only use in SQL connectors, but that's the name for the `map: .` argument in `@relation`, right?

Reply:
Will do that change.

Reply:
I'd like to document what we mean by:

- database name
- mapped name
- foreign key name

In the ParserDatabase docs, so we can have a consistent API. In a PR very soon.